api.planx.uk/pay/proxy.ts

```diff
@@ -8,6 +8,7 @@ export const usePayProxy = (options: Partial<Options>, req: Request) => {
     onProxyReq: fixRequestBody,
     headers: {
       ...(req.headers as NodeJS.Dict<string | string[]>),
+      "content-type": "application/json",
```

Comment:
This is required by GovPay APIs - a `P0920` error response is returned if it's missing. Docs: https://docs.payments.service.gov.uk/api_reference/#gov-uk-pay-api-error-codes

I think it's likely being stripped by one of the proxy layers as the request initially has this, but as it's mandatory it seems to make more sense to set here than pass through.